### PR TITLE
911 display information on geopmap

### LINKED ISSUE #911: Display information on geopmap
show the information when you place the mouse over the area, no need to click to display the information

image

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- Resolve #911 

### Steps to test

1. Run locally, navigate to `http://localhost:3000/rewards-distribution`
2.  hover mouse over countries, daily rewards and paid producers and see popup

### CODE REVIEW

**Comment by fvives99** [comment]
Here's a preview

image

File: webapp/src/components/Tooltip/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import styles from './styles'
 
 const useStyles = makeStyles(styles)
 
-const Tooltip = ({ anchorEl, anchorOrigin, open, onClose, children }) => {
+const Tooltip = ({ anchorEl, anchorOrigin, open, onClose, children}) => {
```
Comment (xavier506) [inline webapp/src/components/Tooltip/index.js:13]: why modify this line?

Comment (fvives99) [reply webapp/src/components/Tooltip/index.js]: It was due to when I was testing if tooltip was the best way to display the `onmouseover` event, I've reverted the changes

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**